sphinx/testing/util.py

```diff
@@ -115,22 +103,9 @@ class SphinxTestApp(application.Sphinx):
     better default values for the initialization parameters.
     """
 
-    def __init__(self, buildername='html', testroot=None, srcdir=None,
+    def __init__(self, buildername='html', srcdir=None,
                  freshenv=False, confoverrides=None, status=None, warning=None,
                  tags=None, docutilsconf=None):
-        if testroot is None:
-            defaultsrcdir = 'root'
-            testroot = rootdir / 'root'
-        else:
-            defaultsrcdir = 'test-' + testroot
-            testroot = rootdir / 'roots' / ('test-' + testroot)
-        if srcdir is None:
-            srcdir = tempdir / defaultsrcdir
-        else:
-            srcdir = tempdir / srcdir
-
-        if not srcdir.exists():
-            testroot.copytree(srcdir)
```

Comment:
Because `tests/root` is moved as `tests/roots/test-root`, path calculation is beome simple